### PR TITLE
[GSoC2024] Displayed total no. of annotations

### LINKED ISSUE #4662: Number of annotation on a single image
Want to increase the number of annotation for viewing a single picture。

### PR DESCRIPTION
**Pull request by KrishavRajSingh**
### Motivation and context
Fixes #4662 
If a person wants to see quickly total no. of annotations, this section may come useful.

### How has this been tested?
image

### Checklist

- [ *] I submit my changes into the `develop` branch
- [ *] I have created a changelog fragment 
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ *] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ *] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: changelog.d/20240322_004121_krishavrajsingh_Count_Annotations.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+### Added <!-- pick one -->
```
Comment (klakhov) [inline changelog.d/20240322_004121_krishavrajsingh_Count_Annotations.md:1]:
```suggestion
### Added
```

File: changelog.d/20240322_004121_krishavrajsingh_Count_Annotations.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+- Added a featire to view all Annaotations at once
```
Comment (klakhov) [inline changelog.d/20240322_004121_krishavrajsingh_Count_Annotations.md:3]:
```suggestion
- Number of objects on the frame is shown on the right sidebar
```

File: changelog.d/20240322_004121_krishavrajsingh_Count_Annotations.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+  (<https://github.com/opencv/cvat/pull/XXXX>)
```
Comment (klakhov) [inline changelog.d/20240322_004121_krishavrajsingh_Count_Annotations.md:4]:
```suggestion
  (<https://github.com/opencv/cvat/pull/7654>)
```

File: cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/objects-list.tsx [hunk 4]
```diff
@@ -79,6 +79,13 @@ function ObjectListComponent(props: Props): JSX.Element {
                 showAllStates={showAllStates}
                 changeShowGroundTruth={changeShowGroundTruth}
             />
+            <Text
+                strong
+                style={{ fontSize: 13 }}
```
Comment (klakhov) [inline cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/objects-list.tsx:84]: Lets move this style to the scss

**Review by klakhov — COMMENTED** [review]
The feature seems to be working, could you add a simple cypress test for it?

**Comment by KrishavRajSingh** [comment]
> The feature seems to be working, could you add a simple cypress test for it?

Hey, it will be good if you refer me some documentation for testing in cvat as I am pretty new to testing.

**Comment by klakhov** [comment]
Hi, you can find e2e tests documentation [here](https://opencv.github.io/cvat/docs/contributing/running-tests/#e2e-tests). You can find lots of cypress test examples [here](https://github.com/opencv/cvat/tree/develop/tests/cypress/e2e). 
Basically  we need to create several objects in the job and test if the sidebar shows correct number.

**Comment by bsekachev** [comment]
1. Are ground truth objects considered in the counter?
2. What do you think regarding adding the same feature for labels and issues? 
3. Will it be possible to add them to Tab headers? 
4. I am not sure "No. of annotations: " is a good option, because abbreviation for "No" is not so obvious

**Comment by KrishavRajSingh** [comment]
> 1. Are ground truth objects considered in the counter?
> 2. What do you think regarding adding the same feature for labels and issues?
> 3. Will it be possible to add them to Tab headers?
> 4. I am not sure "No. of annotations: " is a good option, because abbreviation for "No" is not so obvious

1. Yes, they are considered. What I have done is displayed the length of objectStates. Here is an example: 
[Count_annotation.webm](https://github.com/opencv/cvat/assets/99531396/cd7b3ba4-f9c7-4545-95f1-179c0c03e0ef)
2. I think it is a good idea and am ready to add if you say.
3. Do you mean to replace this here -> 
tabHeader
 ??
4. What is your opinion on "Total annotations"??

**Comment by bsekachev** [comment]
> I think it is a good idea and am ready to add if you say.

Yes, it would be great. 

> What is your opinion on "Total annotations"??

Just "Total count"? 

> Do you mean to replace this here ->

I thought one more time, let's left as is.

**Comment by KrishavRajSingh** [comment]
In this commit I have added "Total Count" for issues and  labels.
[Screencast from 01-04-24 11:40:49 PM IST.webm](https://github.com/opencv/cvat/assets/99531396/442dacb9-a848-46af-8b7c-2cb41eeeeef4)

**Comment by KrishavRajSingh** [comment]
> Hi, you can find e2e tests documentation [here](https://opencv.github.io/cvat/docs/contributing/running-tests/#e2e-tests). You can find lots of cypress test examples [here](https://github.com/opencv/cvat/tree/develop/tests/cypress/e2e). Basically we need to create several objects in the job and test if the sidebar shows correct number.

Hey, I was trying to write some tests but I am not able to login to localhost:8080 in my browser. Localhost:3000 works fine. Where am I doing it wrong??

**Comment by bsekachev** [comment]
`localhost:8080` is default when deploy in docker, `localhost:3000` is default in dev environment. 
You may modify `tests/cypress.config.json` to update application URL

**Comment by bsekachev** [comment]
I will merge this pull request, please send tests as a separate one.
Thank you.

**Pull request merged**